deeppavlov/models/syntax_parser/network.py

```diff
@@ -0,0 +1,352 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from logging import getLogger
+from typing import List, Any, Tuple, Union, Dict
```

Comment:
Any, Tuple, Dict are not used.
The same with array_ops, BertConfig and so on.

You can just run optimize imports in pycharm.

Reply:
Done, thanks for the command.